data/data_121DA0_2.s

```diff
@@ -5,42 +5,8 @@
 
 .section .data
 
-glabel D_802B9140
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B914C
-.word 0x00000000, 0x00000000
-
-glabel D_802B9154
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B9160
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B916C
-.word 0x00000000, 0x00000000
-
-glabel D_802B9174
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B9180
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B918C
-.word 0x00000000, 0x00000000
-
-glabel D_802B9194
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B91A0
-.word 0x00000000, 0x00000000
-
-glabel D_802B91A8
-.word 0x00000000, 0x00000000, 0x00000000
-
-glabel D_802B91B4
-.word 0x00000000, 0x00000000, 0x00000000, 0x0000000d
-.word 0x0000000d
+# I have no idea where this data belongs. It might be some initalized but unused variable?
+.word 0x0000000d, 0x0000000d
```

Comment:
If the previous labels are initialized in code now, did you add this unused variable if that seems to work for now?

Reply:
I don't think there's any direct reference to these values in the code.
D_802B91B4 only takes up 2 words worth of space (its a Vec3s, which due to compiler padding decisions takes up 8 bytes rather the minimum of 6)
Then there's 1 word of 0's that is really the result of file alignment from the linker.
And then there's the mystery double 0xD's.

Reply:
Ah, yea if that's file pad then it would go in the next file. Looks good then!